src/main/resources/seed/admin.sql

```diff
@@ -0,0 +1,37 @@
+
+insert into
```

Comment:
I think we need to seed the database with existing users but this user doesn't exist does it?  Maybe we could move this out of the `src` directory and instead put it in the Github project level under something like "sql"?

Minimum I think we should add a comment at the top of this class clearly explaining why this SQL is in the project.  Unless the system has a tims-admin@smarterbalanced.com user this won't do anything assuming my understanding is correct.

Reply:
The file was moved and comments have been added.  The folder is at the root like you suggested called 'sql'.